magda-web-client/src/actions/feedbackActions.js

```diff
@@ -31,7 +31,6 @@ export function resetFeedback() {
 
 export function fetchFeedback(values) {
     return (dispatch: Function, getState: Function) => {
-        dispatch(sendFeedbacks());
```

Comment:
What's the reason we modified the feedback form logic?

Reply:
@t83714 I think they were merged when @AlexGilleran was working on fixing the failed builds, I'm happy to revert them? They don't need to be changed in this PR

Reply:
@adamzerella 
I think it belongs to PR #823. I simply added a new commit to bring back the changes.
All good now